src/API/users.js

```diff
@@ -0,0 +1,46 @@
+import { client } from '../utils/fetchClient';
+
+
+export const getUsers = async () => {
+  const response = await client.get('/users');
+
+  return response.data;
+};
+
+export const addUser = async (userData) => {
+  const response = await client.post('/users', {
+    name: userData.name,
+    username: userData.name,
+    email: userData.email,
+    phone: "1234567890"
+
+  }, {
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded'
```

Comment:
The `Content-Type` header is set to `application/x-www-form-urlencoded`. Ensure this is the required format for your API. If the API expects JSON, you should use `application/json` instead.